### ISSUE TITLE
Can I use your model to estimate relative depth estimation ?

### ISSUE THREAD

**Issue by YacineDeghaies** [DarthReca/depth-any-canopy#1]
I want to train your model on my own dataset, which has RGBA images + relative depth maps( 8-bit)

Is this possible ? 

Here is an example of my data:

RGB Image:
![shot_0003_source_0000](https://github.com/user-attachments/assets/e304af7d-ca43-4328-86c3-813a2f2719f7)

Ground truth depth map ( 8 bit):
![shot_0003_dm_gt_s_0000](https://github.com/user-attachments/assets/e0d11e32-fb42-450c-a8ee-53079b1c567a)

Thanks in Advance !

**Comment by DarthReca** [comment]
I think the original model (Depth Anything) is a better starting point for this task

**Comment by YacineDeghaies** [comment]
I've tried Depth-Anything-V2 and gives metric depth. What I want is relative depth. But if you mean their original model, there is no training code for it ^^ and am a beginner I can only fine-tune things.

**Comment by DarthReca** [comment]
The base version should be trained for relative depth estimation. If you would like to fine-tune the base model, you can use my code: I have fine-tuned their small and base versions.
